### ISSUE TITLE
Missing Icons -wifi -battery -sound -bt 

### ISSUE THREAD

**Issue by likelazyeyes** [budgie-desktop/budgie-desktop#823]
I removed the default panel and created a new one. But raven's options haven't got wifi and same icons.
I have go to settings every time. This is boring.
Are there a method to restore these?

Solus 2017.01.01.0 64-bit
Budgie 10.2.9-111

**Comment by fossfreedom** [comment]
Have you gone to raven - panel - clicked the + button and have added the status indicator applet?

**Comment by ikeydoherty** [comment]
Wifi is under the tray applet

**Comment by chifreagle** [comment]
Hey @ikeydoherty @fossfreedom ,
I think you guys didn't understand what @likelazyeyes wanted to say, 
he wanted to say that the when he removed his current budgie panel and created a new one,
the new budgie panel didn't keep his current settings, 
if you ask me, that's what should happen, but according to @likelazyeyes the new panel should have the old panel applets and configuration because he doesn't want to go and edit the applets every time he creates a new panel, again I think that not keeping the old applets and the old configaration is a good thing, thanks for you hard work @ikeydoherty

**Comment by ikeydoherty** [comment]
Well its not a bug I know that. I was explaining *where* it is.

TL;DR New panel creates a new, empty panel. Creating a new panel with the same layout each time would completely defeat the purpose of having panels. Closing as non issue.

**Comment by likelazyeyes** [comment]
Okey, i added icons again. Is there is a way to hide/show panel without delete?

Also i can't find power icon: http://i.imgur.com/bfS1Okk.png

**Comment by chifreagle** [comment]
hey @likelazyeyes you can add the power icon by adding the user indicator applet to your budgie panel
to do that you have to open raven ( `super + A` ), then select the configuration icon in the upper right and select "panel options" click on `+` icon and add a user indicator icon (power icon) , that's it :D 
make sure to position your new applet using the arrows ( the panel tab in the budgie setting tab in raven)

**Comment by bomby-x-mori** [comment]
i have thie same problem
